### PR TITLE
Add `dir` prop to Amp and Canonical MenuButton.

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #NUMBER

**Overall change:** _Add `dir` prop to Amp and Canonical MenuButton._

**Code changes:**

- _Add `dir` prop to Amp and Canonical MenuButton._
- _Update snapshots._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by AlistairGempf — APPROVED** [review]
Looks good

**Review by tochwill — APPROVED** [review]

File: packages/components/psammead-navigation/README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ The `@bbc/psammead-navigation` package is a set of two components, `NavigationUl
 | onOpen | function | Yes | N/A | `() => { console.log("Handle open action"); }` |
 | onClose | function | Yes | N/A | `() => { console.log("Handle close action"); }` |
 | isOpen | bool | Yes | N/A | `false` |
+| dir | string | no | `'ltr'` | `'ltr'` |
```
Comment (DenisHdz) [inline packages/components/psammead-navigation/README.md]:
```suggestion
| dir | string | no | `'ltr'` | `'rtl'` |
```

File: packages/components/psammead-navigation/README.md [hunk 2]
```diff
@@ -102,6 +103,7 @@ The `@bbc/psammead-navigation` package is a set of two components, `NavigationUl
 | -------- | ---- | -------- | ------- | ------- |
 | announcedText | string | Yes | N/A | `'Menu'` |
 | onToggle | string | Yes | N/A | `"tap:menu.toggleVisibility"` |
+| dir | string | no | `'ltr'` | `'ltr'` |
```
Comment (DenisHdz) [inline packages/components/psammead-navigation/README.md]:
```suggestion
| dir | string | no | `'ltr'` | `'rtl'` |
```

**Review by DenisHdz — COMMENTED** [review]
We usually put `rtl` in the example.

**Review by DenisHdz — APPROVED** [review]

**Pull request merged**